modules/installation-program-auto-create-subnets.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+
+[id="installation-program-auto-create-subnets_{context}"]
+= Setting the installation program to quickly create subnets for AWS Local Zones
+
+You can set the installation program to automatically create public and private subnets for each AWS Local Zone that you defined in your installation configuration file. By using this installation program in this way, you can quickly install a {product-title} cluster with extended worker nodes that operate in AWS Local Zones locations. 
+
+After you issue the `create cluster` command to start the installation program to deploy your cluster, the installation program completes many tasks. Some of these tasks are described in the following list:
+
+* Creates the machine set manifest for the edge compute pool. 
```

Comment:
SME Query: Do I need to add more bullet points to the list from https://github.com/openshift/enhancements/pull/1232/files#diff-f2590494c9138551ab5b3d5632d11170fe8742a128256f9e1d8ebc63baa19f41R776-R785

Reply:
SME Query: I placed the "Opting into AWS Local Zones" module before the two paths for AWS extending nodes to AWS Local Zones:

* [Install a cluster in existing VPC with Local Zone subnets]
* [Install a cluster setting the Local Zone names on install-config.yaml [full IPI]

However, does the new path for 4.14 provide an auto opt-in capability?